### DISCUSSION TITLE
Can we create a special vdev that is not mirrored?

### DISCUSSION THREAD

**Discussion by Qubitium** [openzfs/zfs#15585] [General]
I would like to to use  a single enterprise level (mlc) with high write endurance for special vdev, mostly for small block offload. However, the zfs cli will not allow me to add it unless it is in mirrored config. 

Is there anyway to force this single non-mirrored vdev?  Or am I doing it wrong? Thanks for any help. 

Ftw, having 2x enterprise ssd for mirror1 is asking for trouble since there is no expected gain in endurance. Unlike spinning disks where the time to failure is more unpredictable.

**Comment by amotin** [comment]
I don't see what is your problem, it even says you how to override it:
```
# zpool create hdd raidz da0p2 da1p2 da2p2 special nvd0p1
invalid vdev specification
use '-f' to override the following errors:
mismatched replication level: both raidz and disk vdevs are present
# zpool create -f hdd raidz da0p2 da1p2 da2p2 special nvd0p1
# zpool status hdd
  pool: hdd
 state: ONLINE
config:

        NAME        STATE     READ WRITE CKSUM
        hdd         ONLINE       0     0     0
          raidz1-0  ONLINE       0     0     0
            da0p2   ONLINE       0     0     0
            da1p2   ONLINE       0     0     0
            da2p2   ONLINE       0     0     0
        special
          nvd0p1    ONLINE       0     0     0

errors: No known data errors
```
But keep in mind, if that single special vdev is lost for any reason, all your pool is lost, since all the metadata is gone with it.

**Reply by Qubitium** [reply]
Thank you. This is an error on my part assuming there must be a parameter present after "special" since almost all ref online uses "special mirror".
